source/features/index.tsx

```diff
@@ -324,7 +324,7 @@ void add('rgh-deduplicator' as FeatureID, {
 		// `await` kicks it to the next tick, after the other features have checked for 'has-rgh', so they can run once.
 		await Promise.resolve();
 		select('#js-repo-pjax-container, #js-pjax-container')?.append(<has-rgh/>);
-		select('#repo-content-pjax-container')?.append(<has-rgh-inner/>); // #4567
+		select('#repo-content-pjax-container, turbo-frame')?.append(<has-rgh-inner/>); // #4567
```

Comment:
I found that `turbo-frame` contains `#repo-content-pjax-container` on the first load and then it's gone. `Select` will return the first element matching the selector, which is always `turbo-frame` since it's the parent